DEVELOP.md

```diff
@@ -0,0 +1,162 @@
+# Development
+
+This document will guide you through how to
+
+. Set up the local development environment
+. Build web UI and configure backend service
+. Run the cloud gaming demo for the local development
+. Add additional games
+
+NOTE: this document assumes that you have installed the [Anbox Cloud Appliance](https://anbox-cloud.io/docs/ref/glossary#anbox-cloud-appliance) on your local machine. Follow the [instructions](https://anbox-cloud.io/docs/tut/installing-appliance) on how to install it if you haven't.
+
+## Web UI
+
+The web UI for the cloud gaming demo is built with [Flutter](https://flutter.dev/).
+To set up the development environment and build the UI, make sure that you have Flutter installed on your machine.
+Follow the [installation instructions](https://docs.flutter.dev/get-started/install/linux) to install it.
+
+### Build the web UI
+
+To build the UI, simply run:
+
+  ```
+  cd ui && flutter build web
+  ```
+
+This builds the UI and places outputs in the `ui/build/web` directory.
+Refer to the official [documentation](https://docs.flutter.dev/get-started/web) for details on how to build web apps with Flutter.
+
+### Deploy the web UI
+
+Create a `static` folder under the `backend/service` directory and copy everything that is in the `ui/build/web` directory into it.
+
+  ```
+  mkdir -p backend/service/static
+  cp -av ui/build/web/* backend/service/static
+  ```
+
+The backend service will then serve those static resources when you access the web UI.
+
+
+## Backend
+
+The backend service does not only serve the web UI, but it also interacts with the Anbox Stream Gateway and provides a set of restful APIs for clients (for example, the web UI) to perform operations like listing games or launching a game.
+
+### Install the dependencies
+
+Before you start, install the `virtualenv` package to create an isolated Python environment in which the backend service can run:
+
+  ```
+  sudo apt-get install python3-pip
+  sudo pip3 install virtualenv
+  cd backend && virtualenv venv
+  ```
+
+### Configure the service
+
+To access the APIs that are exposed by the Anbox Stream Gateway, the backend service must know the following information:
+
+- The URL of the Anbox Stream Gateway
+- The access token for the Anbox Stream Gateway Restful API authentication
+
+As [Anbox Stream Gateway](https://anbox-cloud.io/docs/ref/glossary#stream-gateway) runs behind traefik and secures the Restful API with HTTPS, its URL can be composed by;
```

Comment:
```suggestion
The [Anbox Stream Gateway](https://anbox-cloud.io/docs/ref/glossary#stream-gateway) runs behind traefik and secures the restful API with HTTPS. Specify its URL in the following way:
```

Reply:
I don't understand what we're doing here. This is not a command? And we specify the URL below in the config.yaml file?

Reply:
No, it's not a command. The `gateway_url` here is just linked to the config item(gateway_url) that people need to specify in the config.yaml file. When reading this, it's a bit lengthy and not that clear to me either. Let me improve this or you may have a better idea to describe it in a concise way.